gradle.properties

```diff
@@ -43,6 +43,5 @@ POM_DEVELOPER_URL=https://github.com/slackhq
 POM_INCEPTION_YEAR=2022
 POM_PACKAGING=jar
 SONATYPE_STAGING_PROFILE=com.slack
-SONATYPE_HOST=DEFAULT
 RELEASE_SIGNING_ENABLED=true
```

Comment:
You could also replace this by calling `enableReleaseSigning()` in the DSL

Reply:
assuming this meant `signAllPublications()`?

Reply:
a904f6e

Reply:
Yes  👍